pkgs/by-name/zs/zsh-zhooks/package.nix

```
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub }:
```

```suggestion
{ lib
, stdenv
, fetchFromGitHub
}:
```

Formatted as per RFC 0166 with package `nixfmt-rfc-style`

---

I ran `nix run nixpkgs-unstable#nixfmt-rfc-style -- package.nix` with a fully updated unstable and the comma placement seems different than what you are suggesting. I've left it as it was from me running the tool, but if this is somehow wrong let me know.

---

Good catch, sorry about that!